src/main/java/com/google/devtools/build/lib/remote/RemoteModule.java

```diff
@@ -856,6 +856,10 @@ public void afterCommand() throws AbruptExitException {
 
     if (actionContextProvider != null) {
       actionContextProvider.afterCommand();
+      var gen = actionContextProvider.getTempPathGenerator();
+      if (gen != null) {
+        deleteStaleDownloads(gen.getTempDir());
```

Comment:
The downloads could continue in the background if we enable async downloads. Maybe its not necessary to print the warning message?

Reply:
I wouldn't miss the message, and that's simpler.